common/src/event.rs

```diff
@@ -96,6 +96,23 @@ pub enum Reason<R: RuntimeReason, S: SystemReason> {
 pub enum MessageWaitedRuntimeReason {
     /// Program called `gr_wait` while executing message.
     WaitCalled,
+    /// Program called `gr_wait_for` while executing message.
+    WaitForCalled,
+    /// Program called `gr_wait_up_to` while executing message.
+    WaitUpToCalled,
+    /// Program called `gr_wait_up_to` with enough gas while executing message.
```

Comment:
```suggestion
    /// Program called `gr_wait_up_to` with enough gas for full duration storing while executing message.
```

Reply:
And add similar comment to `WaitUpToCalled` that it has less gas then maximal duration given